public/assets/js/search-calculate.js

```diff
@@ -5,56 +5,69 @@ $(document).ready(function () {
   let periodicChanged = false;
   let initialChanged = false;
 
+  const MOBILE_WIDTH = 576;
+
   $('.search-ready-awesome-eligibility').click(function () {
     $('.search-ready-awesome-div').slideUp(500);
     $('.search-calculate-button').prop('disabled', true);
-    $('.periodic-investment').val('500000');
-    $('.periodic-investment-input').val('500,000');
-    $('.initial-investment').val('100000');
-    $('.initial-investment-input').val('100,000');
+
+    $('.periodic-investment').val('100000');
```

Comment:
Can we make this empty, please? It is a bad UX if I will have to delete the 100000 to add my own value? Or can you explain your thoughts on behind adding these defaults?